tariff/entsoe/static.go

```diff
@@ -32,6 +32,29 @@ type AttributeInstanceComponent struct {
 	AttributeValue string   `xml:"attributeValue"`
 }
 
+type AcknowledgementMarketDocument struct {
```

Comment:
Just to confirm, are you planning on using this type later or should I implement against it?

Reply:
Just added since I had it ready- thought you might want to use it for error handling? Seems that's what gets returned. Thank you!!!